src/global.scss

```diff
@@ -66,6 +61,23 @@ dd {
   }
 }
 
+h3 {
+  font-size: 1.25rem;
+  line-height: 1.4em;
+}
+
+h4 {
+  font-size: 1.125rem;
+  line-height: 1.55556em;
+}
+
```

Comment:
we gebruiken nu geen (r)em waar komen die waardes vandaan?

Reply:
Ik kreeg al `em` en `rem` binnen vanuit de `amsterdam-stijl` package css

Reply:
o ja dat kan kloppen... dan lijkt me dat goed